redis/src/aio.rs

```diff
@@ -573,7 +573,12 @@ where
             self.buf.clear();
             cmd.write_packed_command(&mut self.buf);
             self.con.write_all(&self.buf).await?;
-            self.read_response().await
+            loop {
```

Comment:
please also implement for MultiplexedConnection

Reply:
I'm ok if we leave this as a `TODO` for now (adding explicit comments for such would be helpful) and tackle in a separate PR since `MultiplexedConnection` doesn't currently support pub/sub. I'd like to keep this PR about where it is in terms of changes (though subject to change, I haven't finished reviewing :laughing:)

Reply:
While you're right that MultiplexedConnection doesn't implement pub/sub, that isn't the only situation where PUSH might be used - for example I believe that client-side caching (thanks @altanozlu for starting the conversation) is implemented using that.

Reply:
Makes sense. We'll need additional PRs before releasing these changes but think this particular PR is right sized for now.